snapshot/policy/scheduling_policy_test.go

```diff
@@ -237,8 +254,8 @@ func TestNextSnapshotTime(t *testing.T) {
 		},
 	}
 
-	for i, tc := range cases {
-		t.Run(fmt.Sprintf("case-%v", i), func(t *testing.T) {
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
This becomes part of the test name, which makes it hard to select.

What about changing `name` to description, and logging the description at the beginning of the test (via `t.Log[f]`).

It may be a good idea to still have a `name` for the test, but with something shorter in CamelCase.

WDYT?

Reply:
I didn't think about the ease of selecting a single test.  I think a name is just redundant if t.Log shows up.  Let me try it with the original numeric names and with a log message per test.  I saw it was done like that in the cli policy-set test, but didn't realize why

Reply:
I tried it with t.Log, but it prints all the test logs (not just the failing one). But adding the name to the 'message' field of the require.xxx seems to work really well, so that is what I did.